cl/opinion_page/views.py

```diff
@@ -796,7 +797,15 @@ async def view_opinion_old(
     unbound form.
     """
     # Look up the court, cluster, title and note information
-    cluster: OpinionCluster = await aget_object_or_404(OpinionCluster, pk=pk)
+    cluster: OpinionCluster = await aget_object_or_404(
+        OpinionCluster.objects.prefetch_related(
+            Prefetch(
+                "sub_opinions",
+                queryset=Opinion.objects.order_by("ordering_key"),
```

Comment:
This will work; however I am confused by the general usage of `view_opinion_old`

The ordering is applied for "view_opinion_old", which is called only if `ui_flag_for_o` is not active.
I couldn't find any documentation for `ui_flag_for_o`, but from looking into the waffle admin and seeing it is only active for staff and opt-in users

I think we also want the ordering to work *when* that flag is active?

In general, I think some more comments would help understand what's going on 


-----------

Also, I wanted to test this using some clusters that are out of order without this PR, but couldn't find one. Many opinions with order have their ids in order with the ordering key. Do you have one handy? @flooie

Reply:
This PR is related to 

https://github.com/freelawproject/courtlistener/issues/4378

actually found examples here:

https://github.com/freelawproject/courtlistener/issues/2499

Reply:
So, the ordering for the new UI is done here; so that's not a problem
https://github.com/freelawproject/courtlistener/blob/077192564c5487dc35713afd8094740edb604a62/cl/search/models.py#L2878-L2889

An example of a opinion where the old UI sorts wrong (it sorts by type), and the new UI does it good (by ordering key):
https://www.courtlistener.com/opinion/3472329/a/

By cloning this cluster, and seeing the order we can see that this now has the desired order

`docker exec -it cl-django python /opt/courtlistener/manage.py clone_from_cl --type search.OpinionCluster --id 3472329`

![image](https://github.com/user-attachments/assets/8d39138e-34aa-4ce3-84d4-e3bc8f84a7b2)

The API response also has the desired order
![image](https://github.com/user-attachments/assets/df39c39a-e9d1-4d9e-965d-cb8c2851bf16)

----------------------

So, this is working as expected; my only comment would be to add some comments for the meaning of the flag; and for any differences between the old and the new, including how the ordering is done